LibreNMS/OS/Pepwave.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Pepwave.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2020 Jozef Rebjak
+ * @author     Jozef Rebjak <jozefrebjak@icloud.com>
+ */
+
+namespace LibreNMS\OS;
+
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessSnrDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessRssiDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessSinrDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessRsrpDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessRsrqDiscovery;
+use LibreNMS\OS;
+
+class Pepwave extends OS implements
+    WirelessSnrDiscovery,
+    WirelessRsrpDiscovery,
+    WirelessRsrqDiscovery,
+    WirelessRssiDiscovery,
+    WirelessSinrDiscovery
+{
+    public function discoverWirelessRssi()
+    {
+        $data = snmpwalk_group($this->getDevice(), 'cellularSignalRssi', 'CELLULAR');
+        $sensors = [];
+        foreach ($data as $index => $rssi_value) {
+            if ($rssi_value['cellularSignalRssi'] != '-9999') {
+                $sensors[] = new WirelessSensor('rssi', $this->getDeviceId(), '.1.3.6.1.4.1.23695.200.1.12.1.1.1.3.' . $index, 'pepwave', 'cellularSignalRssi' . $index, 'Celullar ' . ($index+1), null, 1, 1);
```

Comment:
Instead of null, you need to send `$rssi_value['cellularSignalRssi']` for the current value.  This is the same for all the ones below.

Reply:
fixed